electron_app/riot.im/config.json

```diff
@@ -6,6 +6,7 @@
     "integrations_ui_url": "https://scalar.vector.im/",
     "integrations_rest_url": "https://scalar.vector.im/api",
     "integrations_widgets_urls": [
+        "https://scalar-staging.vector.im/api",
```

Comment:
Does the order matter at all? Anyway, looks sane!

Reply:
Looks like the order does not matter, so should be okay.

Reply:
Nope. Thought I would add the new one in the same place as on /app but didn't want to put unnecessary change sin by re-ordering the other ones.

Reply:
Does this really work? Last I knew was that these url overrides don't work currently as per #9476

Reply:
It does work the default file, yes. The issue in #9476 is that we aren't checking alternate files inside the Riot application. (They are only checked at the Electron level, so that's why only update settings apple from an alternate config file at the moment.)